DatadogCrashReporting.podspec

```diff
@@ -0,0 +1,26 @@
+Pod::Spec.new do |s|
+  s.name         = "DatadogCrashReporting"
+  s.version      = "1.20.0"
```

Comment:
1.21.0 maybe?

Reply:
It will be updated when merging `develop` to `feature/v2` 👍

Reply:
Done in #1338